pulp_ansible/app/serializers.py

```diff
@@ -128,6 +139,148 @@ class Meta:
         )
 
 
+class SigstoreSigningServiceSerializer(ModelSerializer):
```

Comment:
This can only be created from the command line, correct? If so then I would change all the fields to `read_only=True`.